### ISSUE TITLE
Data management reports are not accessible via this method

### ISSUE THREAD

**Issue by kkinkopf** [MicrosoftDocs/CloudAppSecurityDocs#322]
The data management reports are no longer listed under the three vertical dots on the right page. How do you generate these reports in the new portal? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ba61c236-e64a-2835-8fbd-4ff975e78d27
* Version Independent ID: d4394353-4330-05a2-c30b-b2b4bd7b7c2f
* Content: [Generate reports - Microsoft Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/built-in-reports)
* Content Source: [CloudAppSecurityDocs/built-in-reports.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/master/CloudAppSecurityDocs/built-in-reports.md)
* Service: **cloud-app-security**
* Product: ****
* Technology: ****
* GitHub Login: @shsagir
* Microsoft Alias: **shsagir**

**Comment by shsagir** [comment]
@kkinkopf Thanks for contacting Microsoft about your issue. As we understand it, you're describing a problem with the product itself and not with the product documentation.

To get assistance or support for your product issue, please [open a support ticket](https://go.microsoft.com/fwlink/?linkid=2099672) or [Send feedback](mailto:casfeedback@microsoft.com).

**Comment by kkinkopf** [comment]
Hello Shlomo,

I disagree. I have checked on four (4) clients since I lost wrote this and they cannot access those reports in this fashion. I suspect you are using an old tenant for your review. I believe this feature has been removed and now can only be accessed in a different format as a query. This document should be updated as Data Management Reports are not accessible making the documentation incorrect.
